### ISSUE TITLE
Provide time unit for “Update interval” in preferences

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#52]
```
What steps will reproduce the problem?
1. Go to Preferences, tab “Main”.

What is the expected output? What do you see instead?
There should be a unit for the update interval. I only see the number 60. Is 
this in seconds or in minutes?

What version of the product are you using? On what operating system?
Pino 0.1.3,
Linux Mint Helena.

```

Original issue reported on code.google.com by `alex.bepple` on 23 Jan 2010 at 3:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 24 Jan 2010 at 12:03
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 29efa0396b revision.
```

Original comment by `tro...@gmail.com` on 31 Jan 2010 at 12:36
- Changed state: **Fixed**
